### ISSUE TITLE
Category not updating during CSV Import

### ISSUE THREAD

**Issue by jmochnick** [snipe/snipe-it#3309]
#### Expected Behavior (or desired behavior if a feature request)

When I import assets from a CSV, the Category field should be updated during the import.

-----

#### Actual Behavior

All fields import correctly, but the Category field does not.

-----

#### Please confirm you have done the following before posting your bug report:

- [ x ] I have enabled debug mode
- [ x ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Please provide answers to these questions before posting your bug report:

- v3.6.3 build 15 (g1bf34d7)
- RHEL 6.7, Apache
- install.sh
- No errors
- Import is disabled in demo

So basically, I did a few imports, and we soon after realized that the categories were incorrect. So, I corrected the CSV import file with the correct category names, did the import, and made sure to check the box to update the assets that are already created. Other fields will update but the category will not.

**Comment by snipe** [comment]
What's your field name for category in yur CSV?

**Comment by jmochnick** [comment]
It's just Category with no spaces before or after.

**Comment by snipe** [comment]
Did you ever find a resolution to this issue? I don't think anyone else has reported this problem, so will close if you have.

**Comment by KeenRivals** [comment]
I'm running into this issue as well. As a curiosity, do you have models that you're trying to put in different categories? For example, I have ThinkPad Yoga 260 models, but I want some in "Student Laptops" and others in "Employee Laptops". It seems like I can only assign *models* to categories, not *assets*.

**Comment by snipe** [comment]
@KeenRivals categories are assigned at the model level, not the asset level.
